src/core/AutoRest.Core/Properties/Resources.resx

```diff
@@ -251,10 +251,10 @@ Modelers:
     <value>For the tracked resource '{0}', the x-ms-pageable extension values must be same for list by resource group and subscriptions operations.</value>
   </data>
   <data name="TrackedResourceListByResourceGroupOperationMissing" xml:space="preserve">
-    <value>The tracked resource, '{0}', must have a list by resource group operation.</value>
+    <value>The tracked resource, '{0}', must have a list by resource group operation.(This rule does not apply for tenant level resources.)</value>
```

Comment:
Can we detect tenant level resource from the `path` of the operation to improve the logic of the rule?